iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -826,7 +826,16 @@ IOTHUB_CLIENT_RESULT IoTHubClient_LL_UploadMultipleBlocksToBlob_Impl(IOTHUB_CLIE
                                         else
                                         {
                                             /*must make a json*/
-                                            STRING_HANDLE req_string = STRING_construct_sprintf("{\"isSuccess\":%s, \"statusCode\":%d, \"statusDescription\":\"%s\"}", ((httpResponse < 300) ? "true" : "false"), httpResponse, BUFFER_u_char(responseToIoTHub));
+                                            unsigned char * tmp = BUFFER_u_char(responseToIoTHub);
+                                            STRING_HANDLE req_string;
+                                            if(tmp == NULL)
```

Comment:
it might be more elegant to set the value for temp and check if it is null in the same line via `if((tmp = BUFFER_u_char(responseToIoTHub)) == NULL)`

Reply:
I find it more readable the way it is, however renamed tmp.